gradle-conjure/src/main/java/com/palantir/gradle/conjure/CompileIrTask.java

```diff
@@ -67,17 +69,28 @@ public final File getExecutableDir() {
 
     @TaskAction
     public final void generate() {
-        getProject().exec(execSpec -> {
-            ImmutableList.Builder<String> commandArgsBuilder = ImmutableList.builder();
-            commandArgsBuilder.add(
-                    new File(executableDir.get(), EXECUTABLE).getAbsolutePath(),
-                    "compile",
-                    inputDirectory.get().getAbsolutePath(),
-                    outputFile.getAbsolutePath());
+        ByteArrayOutputStream output = new ByteArrayOutputStream();
 
-            List<String> args = commandArgsBuilder.build();
+        ImmutableList.Builder<String> commandArgsBuilder = ImmutableList.builder();
+        commandArgsBuilder.add(
+                new File(executableDir.get(), EXECUTABLE).getAbsolutePath(),
+                "compile",
+                inputDirectory.get().getAbsolutePath(),
+                outputFile.getAbsolutePath());
+
+        List<String> args = commandArgsBuilder.build();
+
+        ExecResult execResult = getProject().exec(execSpec -> {
             getLogger().info("Running compiler with args: {}", args);
             execSpec.commandLine(args.toArray());
+            execSpec.isIgnoreExitValue();
+            execSpec.setStandardOutput(output);
```

Comment:
Can we capture stderr here as well?

Reply:
This is both stdout and stderr

Reply:
ha overloading :-) Thanks

Reply:
Was confused, this is actually just stdout like you thought. Now I'm just doing stderr.

Reply:
Any reason not to write both to our output buffer?